t3-oss/create-t3-turbo
# Fix/nativewind ts config

# Tailwind typescript config file for expo/nativewind.

This PR allows expo and nativewind to use `tailwind.config.ts` configuration just like the NextJS app.

## How to test

1. Run `pnpm install`
2. Start the emulator and run `pnpm dev` 
3. Make sure babel compiles and tailwind configuration is applied.

4. (Optional) You might need to clear the cache
https://docs.expo.dev/troubleshooting/clear-cache-macos-linux/
https://docs.expo.dev/troubleshooting/clear-cache-windows/
